README.md

```diff
@@ -269,7 +275,7 @@ const article = new Schema('articles', { defaults: { likes: 0 } });
 
 ### `Schema.prototype.define(nestedSchema)`
 
-Lets you specify relationships between different entities.  
```

Comment:
Please preserve trailing whitespace in markdown. Two spaces at the end of a line indicates a line break.